regression/pages/studio/pages_page_studio.py

```diff
@@ -0,0 +1,69 @@
+"""
+Pages page
+"""
+from edxapp_acceptance.pages.common.utils import (
+    wait_for_notification,
+    click_css)
+
+from regression.pages.studio.course_page_studio import CoursePageExtended
+
+
+class PagesPageExtended(CoursePageExtended):
+    """
+    Pages page.
+    """
+    url_path = "tabs"
+
+    def is_browser_on_page(self):
+        return self.q(css='body.view-static-pages').visible
+
+    def add_page(self):
+        """
+        Adds a new empty page.
+        """
+        click_css(self, '.button.new-button.new-tab', 0, False)
+        self.wait_for_element_visibility(
+            '.component.course-tab.is-movable', 'New page is not visible')
+
+    def edit_page(self, new_content, index=0):
+        """
+        Edits the page present at index passed.
+        """
+        click_css(self, '.action-button-text', index, False)
+        self.browser.execute_script(
+            'tinyMCE.activeEditor.setContent("{}")'.format(new_content))
+        self.browser.execute_script(
+            'document.querySelectorAll(".button.action-primary'
+            '.action-save")[0].click();')
+        wait_for_notification(self)
+
+    def delete_page(self, index=0):
+        """
+        Delete the page present at index passed.
+        """
+        click_css(self, '.delete-button.action-button', index, False)
+        self.q(css='.prompt.warning button.action-primary ').first.click()
+        wait_for_notification(self)
+
+    def reload_and_wait_for_page(self):
+        """
+        Reloads and waits for the newly added
+        page to appear.
+        """
+        self.browser.refresh()
+        self.wait_for_element_visibility(
+            '.delete-button.action-button', 'Added pages has been loaded.')
+
+    def get_page_count(self):
+        """
+        Get the count of all new pages added.
+        """
+        return len(self.q(css='.component.course-tab.is-movable'))
+
+    def get_page_content(self, index=0):
+        """
+        Get the contents of a page present at index passed.
+        """
+        click_css(self, '.action-button-text', index, False)
+        return self.browser.execute_script(
+            'return tinyMCE.activeEditor.getContent()')
```

Comment:
Also break the long lines in this file according to http://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python_guidelines.html#breaking-long-lines